backends/tfhe-cuda-backend/build.rs

```diff
@@ -54,8 +54,7 @@ fn main() {
         }
 
         if cfg!(feature = "debug") {
-            cmake_config.define("CMAKE_BUILD_TYPE", "DEBUG");
-            cmake_config.define("CMAKE_CXX_FLAGS", "-Wuninitialized -O0");
+            cmake_config.define("CMAKE_BUILD_TYPE", "Debug");
```

Comment:
I also just stumbled upon this issue, the string is case sensitive compared to the "Debug" literal string so my previous approach wasn't actually working . I will add code to make the comparison case insensitive in another PR

Reply:
Ok thanks, in the meanwhile this version is doing what we want, at least I could ensure the noise asserts are triggered when the debug feature is activated.